tests/scripts/github-action-helper.sh

```diff
@@ -561,7 +566,7 @@ function write_object_read_from_replica_cluster() {
   # a direct sub-shell.
   S3CMD_ERROR=0
   (
-    sleep 60
+    sleep 300
```

Comment:
Is this the effective change that helps stabilize the test? 😅

Reply:
Yes/no/maybe?  It was initially extended to 600s, which seemed to work. I am hopeful that the new waits on the `cephcluster` and `cephobjectstore` CRs would make it unnecessary to wait so long for object replication to work. I did see a failure when trying to change it back to 60s.  My thinking is we would set it a little long and then try to tighten it up.

Reply:
Got it! I'm glad to have any improvement in stability here. It's been a long time needed.

Reply:
On the most recent run in this PR, it took 94s.  I suspect there is significant jitter.